holmes/plugins/toolsets/datadog/toolset_datadog_metrics.py

```diff
@@ -238,17 +256,60 @@ def _invoke(self, params: Any) -> StructuredToolResult:
                     params=params,
                 )
 
+            # Transform Datadog series data to match Prometheus format
+            prometheus_result = []
+            for serie in series:
+                # Extract metric info from Datadog series
+                metric_info = {}
+                if "metric" in serie:
+                    metric_info["__name__"] = serie["metric"]
+
+                # Add other fields from scope/tag_set if available
+                if "scope" in serie and serie["scope"]:
+                    # Parse scope like "pod_name:robusta-runner-78599b764d-f847h" into labels
+                    scope_parts = serie["scope"].split(",")
+                    for part in scope_parts:
+                        if ":" in part:
+                            key, value = part.split(":", 1)
+                            metric_info[key.strip()] = value.strip()
+
+                # Transform pointlist to values format (timestamp, value as strings)
+                values = []
+                if "pointlist" in serie:
+                    for point in serie["pointlist"]:
+                        if len(point) >= 2:
+                            # Convert timestamp from milliseconds to seconds, format as string
+                            timestamp = int(point[0] / 1000)
+                            value = str(point[1])
+                            values.append([timestamp, value])
+
+                prometheus_result.append({"metric": metric_info, "values": values})
+
+            # Convert timestamps to RFC3339 format for start/end
+            from datetime import datetime
```

Comment:
Any reason that we keep this import here?

Reply:
fixed